__tests__/app.test.js

```diff
@@ -163,13 +164,155 @@ test("200: responds with updated votes", () => {
+  
+  test("200: responds with correct content keys", () => {
+   return request(app)
+    .get("/api/articles")
+    .expect(200)
+    .then((response) => {
+      let {
+        _body: articles
+      } = response
+      articles = articles.article
+          articles.forEach((article) => {
+            expect(article).toEqual(
+              expect.objectContaining({
+                article_id: expect.any(Number),
+                title: expect.any(String),
+                topic: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                comment_count: expect.any(Number),
+              })
+            )
+           })
+          })
+        })
+        
+  test("200: responds with articles in descending order", () => {
+    return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response)=> {
+          const {
+            _body: { article },
+          } = response;
+          expect(article).toBeSortedBy(`created_at`, {
+            descending: true
+          })
+        })
+      })
+      
     })
-})
-})
\ No newline at end of file
+
+    describe("9. GET: /api/articles/:article_id/comments", () => {
+      test("200: responds with correct (updated) article with comments_count", () => {
+       return request(app)
+        .get("/api/articles/5/comments")
+        .expect(200)
+        .then((response) => {
+          const {
+            body: { article },
+          } = response;
+          expect(article).toEqual([
+            {
+              comment_id: 15,
+              body: "I am 100% sure that we're not completely sure.",
+              author: 'butter_bridge',
+              created_at: '2020-11-24T00:08:00.000Z',
+              votes: 1
+            },
+            {
+              comment_id: 14,
+              body: 'What do you see? I have no idea where this will lead us. This place I speak of, is known as the Black Lodge.',
+              author: 'icellusedkars',
+              created_at: '2020-06-09T05:00:00.000Z',
+              votes: 16
+            }
+          ])
+        })
+      })
+
+      test("200: responds with the comments in descending order by date", () => {
+        return request(app)
+         .get("/api/articles/5/comments")
+         .expect(200)
+         .then((response) => {
+           const {
+             body: { article },
+           } = response;
+           expect(article).toBeSortedBy(`created_at`, {
+             descending: true
+           })
+         })
+       })
+       
+       test("404: responds with error for a route that is not available although is valid", () => {
+        return request(app)
+          .get("/api/articles/200/comments")
+          .expect(404)
+          .then(({body})=> {
+            expect(body.msg).toBe("This article does not exist")
+          })
+    })
```

Comment:
You will also need another tests for when an article is valid, but has no comments on it. This would be a 200 status code.